StandardConfig/production/CaloDigi/SiWEcalDigi.xml

```diff
@@ -45,7 +45,7 @@
     <parameter name="CellIDLayerString"> layer </parameter>
     <parameter name="CellIDModuleString"> module </parameter>
     <parameter name="CellIDStaveString"> stave </parameter>
-    <parameter name="expectedInterModuleDistance"> 7.0 </parameter>
```

Comment:
You should probably keep this parameter. For documentation and in case we want to switch ON the module correction, this parameter should be set with the correct value

Reply:
OK, done